lib/php/rendition-factory/src/Config/YamlLoader.php

```diff
@@ -60,9 +60,7 @@ private function parseFamilyConfig(array $data): FamilyBuildConfig
 
         $transformations = [];
         foreach ($data['transformations'] as $transformation) {
-            if ($transformation['enabled'] ?? true) {
-                $transformations[] = $this->parseTransformation($transformation);
-            }
+            $transformations[] = $this->parseTransformation($transformation);
```

Comment:
Is it because you also want to validate disabled transformations? I don't think we want this: you could want to disable a transformation because my options are not properly set yet.

Reply:
The idea was to check the options for a transformation, even if it is disabled (and enabling when check is ok).
During rendition creation, disabled transformations are fully ignored, so an buggy setting is only reported while cli "checking" conf.

Reply:
OK